sql_generators/glean_usage/baseline_clients_daily.py

```diff
@@ -1,5 +1,5 @@
 """Generating and run baseline_clients_daily queries for Glean apps."""
-from bigquery_etl.glean_usage.common import GleanTable
+from common import GleanTable
```

Comment:
This file and many following it have permissions changed from `644` to `755` (executable) in this PR. Was that change intentional?

Reply:
Mh interesting, changed it back. I thought that maybe the way these files are added as part of the CLI might have caused a permission change, but testing it locally that doesn't seem to be the case. So not sure what happened, but should be fixed now